packages/create-burner/src/manager/burnerManager.ts

```diff
@@ -305,6 +338,11 @@ export class BurnerManager {
             }
         );
 
+        // shouldn't we wait to make sure it was accepted?
```

Comment:
yes we should wait - can you include this

Reply:
done in https://github.com/dojoengine/dojo.js/pull/186/commits/cb3f72d9285c32819db1c7fb962a0a7cf4c826e6